user-manual/transfer/import-metadata.rst

```diff
@@ -93,15 +93,15 @@ for creating a metadata.csv file that will apply metadata to individual objects.
 #. Create a CSV file called metadata.csv. The example below can be used as
    a template for Dublin Core metadata. The filename column must come first and
    the filename path must always start with ``objects/``. Note that if you add
-   column headings that are not in the ``dc.element`` format, Archivematica will
-   not indicate that the metadata is Dublin Core.
+   column headings that are not in the ``dc.element`` or ``dcterms.element``
```

Comment:
Good catch!

Reply:
I confess the issue 670 pointed me in the right direction!